frontend/src/bundles/common/components/bar-chart/bar-chart.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+    BarElement,
+    CategoryScale,
+    Chart as ChartJS,
+    LinearScale,
+    Tooltip,
+} from 'chart.js';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+
+import styles from './bar.module.css';
+import {
+    INCOME_COLOR,
+    INCOME_LABEL,
+    OUTCOME_COLOR,
+    OUTCOME_LABEL,
+} from './bar-constants';
+import { options } from './config-bar';
+import { dateHelper } from './datebar-helper';
+import { type ChartProperties } from './types-bar';
+
+ChartJS.register(CategoryScale, LinearScale, BarElement, Tooltip);
+
+const Chart: React.FC<ChartProperties> = ({ income, outcome }) => {
+    const data = {
+        datasets: [
+            {
+                label: OUTCOME_LABEL,
+                data: dateHelper(outcome),
+                backgroundColor: OUTCOME_COLOR,
+            },
+            {
+                label: INCOME_LABEL,
+                data: dateHelper(income),
+                backgroundColor: INCOME_COLOR,
+            },
+        ],
+    };
+
+    return <Bar options={options} data={data} className={styles.chart} />;
+};
+
+Chart.propTypes = {
```

Comment:
we don't use prop types.

Reply:
fixed: [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/32/commits/041e22d21b1ee608d88d1097227684812d5c9000](url)